Tests/Functional/Domain/Repository/Product/TeaRepositoryTest.php

```diff
@@ -66,6 +66,15 @@ public function findAllSortsByTitleInAscendingOrder(): void
         self::assertSame(2, $result->current()->getUid());
     }
 
+    /**
+     * @test
+     */
+    public function findByUidForNoneExistingRecord(): void
```

Comment:
Let's make the test name a full sentence containing a statement about the checked behavior:

```suggestion
    public function findByUidForInexistentRecordReturnsNull(): void
```

https://speakerdeck.com/oliverklee/test-driven-development-with-phpunit-915a5f2c-3d37-4e41-b5bc-ef1efca9c01e?slide=39